tests/compile_test.py

```diff
@@ -51,6 +52,7 @@ def test_atomic_swap():
 &&
 ||
 &&"""
+    reset_label_count()
```

Comment:
Out of curiosity, why are these required here?

Reply:
Those are required in general to reset the global label counter in `util.py` so that the labels match the expected code string. Although it looks like this example does not actually use labels, so that call can be removed from here if you think it's confusing.

Reply:
Ah, okay--that makes sense.

I don't think it's necessary to remove them; rather, it seems somewhat redundant that it's necessary to place it before every test for the tests to be correct.

To a larger point, it's a little unfortunate that state for these labels is global and not already implicitly reset per invocation of `compileTeal`, which is what intuitively makes sense to me. Perhaps there's a way to move this global counter to somewhere local to `compileTeal`.